docs/api.md

```diff
@@ -1200,7 +1200,7 @@ GET /api/roles
 <a name='new-role'></a>
 ### Add a new role [POST /api/roles]
 + requires [permission](#permissions) `manageRoles`
-+ `name` (string) - Max length 32.
++ `name` (name) - Max length 32.
```

Comment:
Noooo

This was deliberate - using `name` here means people can't call their roles `cool people` or anything because it has to fit the name regex.